### PR TITLE
Add lymphoma branch DEV SNAPSHOT (HPO)

### PR DESCRIPTION
**Pull request by matentzn**
cc @pnrobison

### CODE REVIEW

File: config/ols-config/ols-config.yaml [hunk 1]
```diff
@@ -81,7 +95,7 @@ ontologies:
     oboSlims: false
     ontology_purl : file:/mnt/ontologies/mondo-rare.owl
   - id: mondo
-    preferredPrefix: MONDO
+    preferredPrefix: MONDO_DEV
```
Comment (twhetzel) [inline config/ols-config/ols-config.yaml:98]: in OLS, does this change the term IRIs?

File: docker-compose.yml [hunk 2]
```diff
@@ -31,15 +31,16 @@ services:
...
         - OLS_HOME=/mnt/
         - ols.customisation.debrand=true
-        - ols.customisation.logo=/custom/logo.png
-        - ols.customisation.title=OLS Instance
-        - ols.customisation.org=CUSTOM
-        - ols.customisation.short-title=CUSTOM-OLS
+        - ols.customisation.logo=https://raw.githubusercontent.com/jmcmurry/closed-illustrations/master/logos/monarch-logos/monarch-logo-white.png
+        - ols.customisation.title=Ontology Lookup Service
+        - ols.customisation.org=Monarch Initiative
+        - ols.customisation.short-title=MOLS
```
[the remark this answers is not in the document]
Comment (twhetzel) [reply docker-compose.yml:39]: Can you also update `ols.customisation.supportMail` to something other than EBI's support email address? Error  pages show the support email address: https://ols.monarchinitiative.org/ontologies/mondo/download

**Review by twhetzel — COMMENTED** [review]
Question/comments inline.

Was this intended to be merged into `EBISPOT:master`?

**Comment by matentzn** [comment]
Ahhhh noooo, of course not!! Here stupid GitHub.. sorry sorry it was supposed to go to our fork!

File: config/ols-config/ols-config.yaml [hunk 3]
```diff
@@ -1,37 +1,142 @@
...
+    base_uri:
+      - http://purl.obolibrary.org/obo/MONDO_
+    synonym_property:
+      - http://www.geneontology.org/formats/oboInOwl#hasExactSynonym
+    definition_property:
+      - http://purl.obolibrary.org/obo/IAO_0000115
+#    label_property:  http://www.w3.org/2004/02/skos/core#prefLabel
+    reasoner: EL
+    oboSlims: false
+    ontology_purl : file:/mnt/ontologies/mondo.owl
```
[the remark this answers is not in the document]
Comment (matentzn) [reply config/ols-config/ols-config.yaml]: No this is generated by the makefile right in place before OLS is loaded.

File: config/ols-config/ols-config.yaml [hunk 4]
```diff
@@ -1,37 +1,142 @@
...
+#    label_property:  http://www.w3.org/2004/02/skos/core#prefLabel
+    reasoner: EL
+    oboSlims: false
+    ontology_purl : file:/mnt/ontologies/hp.owl
+  - id: mondo
+    preferredPrefix: MONDO
+    title: Mondo Disease Ontology (Development Snapshot)
+    uri: http://purl.obolibrary.org/obo/mondo.owl
+    description: >
+      Inofficial development snapshot of the Mondo Disease Ontology. Contact Nicole Vasilevsky.
```
[the remark this answers is not in the document]
Comment (matentzn) [reply config/ols-config/ols-config.yaml]: Done. thx

File: docker-compose.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (matentzn) [reply docker-compose.yml:39]: Didnt know that, thanks! Done.

File: config/ols-config/ols-config.yaml [hunk 5]
```diff
@@ -1,37 +1,142 @@
...
+      - http://purl.obolibrary.org/obo/HP_
+    synonym_property:
+      - http://www.geneontology.org/formats/oboInOwl#hasExactSynonym
+    definition_property:
+      - http://purl.obolibrary.org/obo/IAO_0000115
+#    label_property:  http://www.w3.org/2004/02/skos/core#prefLabel
+    reasoner: EL
+    oboSlims: false
+    ontology_purl : file:/mnt/ontologies/hp.owl
+  - id: mondo
```
[the remark this answers is not in the document]
Comment (matentzn) [reply config/ols-config/ols-config.yaml]: We dont add the latest release here.. just the development snapshots! The latest release is in OLS!

File: config/ols-config/ols-config.yaml [hunk 6]
```diff
@@ -62,11 +62,25 @@ ontologies:
     definition_property:
       - http://purl.obolibrary.org/obo/IAO_0000115
 #    label_property:  http://www.w3.org/2004/02/skos/core#prefLabel
-    reasoner: EL
     oboSlims: false
     ontology_purl : file:/mnt/ontologies/hp-edit.owl
+  - id: hp
```
[the remark this answers is not in the document]
Comment (matentzn) [reply config/ols-config/ols-config.yaml:67]: Good catch, fixed.

File: config/ols-config/ols-config.yaml (hunk 1 above)
Comment (matentzn) [reply config/ols-config/ols-config.yaml:98]: No, this is just used on the ontologies overview page.

**Pull request closed without merging**